src/Angular.js

```diff
@@ -893,7 +894,53 @@ var csp = function() {
   return (csp.isActive_ = active);
 };
 
+/**
+ * @ngdoc directive
+ * @name ngJq
+ *
+ * @element html
+ * @param {string=} the name of the library available under `window`
+ * to be used for angular.element
+ * @description
+ * Use this directive to force the angular.element library.  This should be
+ * used to force either jqLite by leaving ng-jq blank or setting the name of
+ * the jquery variable under window (eg. jQuery).
+ *
+ * Since this directive is global for the angular library, it is recommended
+ * that it's added to the top-most HTML element, but it is not mandatory.
+ *
+ * @example
+ * This example shows how to force jqLite using the `ngJq` directive to the `html` tag.
+ ```html
+ <!doctype html>
+ <html ng-app ng-jq>
+ ...
+ ...
+ </html>
+ ```
+ * @example
+ * This example shows how to use a jQuery based library of a different name.
+ * The library name must be available at the top most 'window'.
+ ```html
+ <!doctype html>
+ <html ng-app ng-jq="jQueryLib">
+ ...
+ ...
+ </html>
+ ```
+ */
+var jq = function() {
+  if (isDefined(jq.name_)) return jq.name_;
 
+  var el = document.querySelector('[ng-jq]') || document.querySelector('[data-ng-jq]');
+  var name;
+
+  if (el) {
+    name = el.getAttribute('ng-jq') || el.getAttribute('data-ng-jq') || '';
```

Comment:
I know (`ngApp` isn't a regular directive either). Yet, `ngCsp` shouldn't be that common; I would expect `ngJq` to be more popular.

In any case, since they are presented as directives in the docs, I would either expect them to conform to the directive normalization rules or have an explicit notice about the recognized forms.

<sub>(This is also true for `ngApp`/`ngCsp`, not only `ngJq`.)</sub>